status.php

```diff
@@ -195,10 +195,15 @@
    }
 
    // Check crontask
-   $crontasks = getAllDatasFromTable('glpi_crontasks',
-                                     "`state`=".CronTask::STATE_RUNNING."
-                                      AND ((unix_timestamp(`lastrun`) + 2 * `frequency` < unix_timestamp(now()))
-                                           OR (unix_timestamp(`lastrun`) + 2*".HOUR_TIMESTAMP." < unix_timestamp(now())))");
+   $crontasks = getAllDatasFromTable(
+      'glpi_crontasks', [
+         'state'  => CronTask::STATE_RUNNING,
+         'OR'     => [
+            new \QueryExpression('(unix_timestamp(`lastrun`) + 2 * `frequency` < unix_timestamp(now()))'),
+            new \QueryExpression('(unix_timestamp(`lastrun`) + 2*".HOUR_TIMESTAMP." < unix_timestamp(now()))')
```

Comment:
HOUR_TIMESTAMP concatenation seems to use bad quote, i think using `'` instead `"` will be better